library/cwm/src/lib/cwm/common_widgets.rb

```diff
@@ -271,6 +271,8 @@ def cwm_definition
 
   # A selection of a value via radio buttons.
   # The {#label} method is mandatory.
+  # @note if radio buttons are modified during runtime, like with #change_items
+  #   then handle won't work correctly unless handle_all_events specified
```

Comment:
no, actually it is such fix handling events. this note is that automatic id registration of buttons used in RadioButtons Group won't work if buttons was later changed, unless it is registered to get all events.